### PR TITLE
Add sidebar for project selection

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -9,7 +9,7 @@ http {
   }
 
   server {
-    listen 5000;
+    listen 9000;
```
Comment (acrellin) [inline nginx.conf]: Why change this?

File: package.json [hunk 2]
```diff
@@ -10,19 +10,21 @@
     "react-dom": "^15.1.0",
     "react-modal": "latest",
     "react-tabs": "latest",
+    "react-dropzone": "latest",
```
Comment (acrellin) [inline package.json]: We're not using these anymore though, right?

File: nginx.conf (hunk 1 above)
Comment (bnaul) [inline nginx.conf]: I think that was an accident, he said he has to run on 9000 locally.

File: package.json [hunk 3]
```diff
@@ -10,19 +10,21 @@
...
     "bootstrap-css": "latest",
+    "underscore": "latest",
     "filter-values": "latest",
+    "react-checkbox-group": "latest",
     "redux": "latest",
     "react-redux": "latest",
     "redux-thunk": "latest",
     "redux-logger": "latest",
-    "whatwg-fetch": "latest",
-    "redux-form": "latest",
```
Comment (acrellin) [inline package.json:24]: Is your branch up to date? We need these packages...

**Comment by acrellin** [comment]
Nice CSS wizardry! Some of these changes seem to be accidental though

**Comment by stefanv** [comment]
Sorry about that, fixed

**Pull request merged**